Userland/Libraries/LibCrypto/Hash/MGF.h

```diff
@@ -23,9 +23,11 @@ class MGF {
 
         size_t h_len = hash.digest_size();
 
+#ifdef __LP64__
```

Comment:
LP64 isn't quite enough to select "size_t is > 32 bits" . For example, the MSVC ABI is LLP64. Surely we have AK_IS_64_BIT? or similar...

or we could do like to be explicit about the check.

```
if constexpr (sizeof(size_t) > 32)  {
   if (length > (h_len << 32)
       return ...;
}
```

Reply:
> `if constexpr`

Will that get rid of the compile time error though?

Reply:
yes <https://godbolt.org/z/qc5xqWro1>

Reply:
Done